pkg/occlient/occlient_test.go

```diff
@@ -602,6 +603,21 @@ func TestCreateRoute(t *testing.T) {
 			existingDC: *fakeDeploymentConfig("blog", "", nil, nil, t),
 			secureURL:  true,
 		},
+
+		{
+			name:       "Case : specify a path",
```

Comment:
Weird case name. I think you forgot to specify a number? For ex. `Case 1: Specify a path` or something?

Reply:
Non of the test cases in `TestCreateRoute` has a case number defined, I'm following the format

Reply:
Cases don't need to have a number. More important is to have good description.